# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libowfat/dietlibc.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- libowfat-0.19.2.orig/GNUmakefile	2004-03-27 20:49:35.000000000 -0500
+++ libowfat-0.19.2/GNUmakefile	2004-05-11 00:16:53.000000000 -0400
@@ -23,24 +23,6 @@
 CFLAGS=-pipe -Wall -g
 endif
 path = $(subst :, ,$(PATH))
-diet_path = $(foreach dir,$(path),$(wildcard $(dir)/diet))
-ifeq ($(strip $(diet_path)),)
-ifneq ($(wildcard /opt/diet/bin/diet),)
-DIET=/opt/diet/bin/diet
-else
-DIET=
-endif
-else
-DIET:=$(strip $(diet_path))
-endif
-
-ifneq ($(DIET),)
-ifneq ($(DEBUG),1)
-DIET+=-Os
-endif
-endif
-# to build without diet libc support, use $ make DIET=
-# see http://www.fefe.de/dietlibc/ for details about the diet libc
 
 VPATH=str:byte:fmt:scan:uint:open:stralloc:unix:socket:buffer:mmap:textcode:taia:tai:dns:case:array:mult:io
 
